syntax = "proto3";
package first;

message CountRequest{
 string id=1;
}

message CountResponse {
 int32 count=1;
}


service Counter {
 rpc getCount(CountRequest) returns (CountResponse){}
}


